etl-pipeline/processes/grin/conversion.py

```diff
@@ -164,19 +168,17 @@ def _sync_converted_books(self) -> set:
                     .all()
                 )
 
-                if not barcodes_to_update:
-                    continue
```

Comment:
@jackiequach I'm not sure that I'm understanding the reason for this continue, and don't want to be bypassing it if its necessary!